packages/sales-api-service/src/services/renewals/__tests__/renewals.service.spec.js

```diff
@@ -107,7 +136,42 @@ describe('preparePermissionDataForRenewal', () => {
         licenceStartTime: endDate.hours(),
         renewedEndDate: endDate.toISOString()
       }
-      expect(preparePermissionDataForRenewal(existingPermission({ endDate }))).toEqual(expect.objectContaining(expectedData))
+      expect(await preparePermissionDataForRenewal(existingPermission({ endDate }))).toEqual(expect.objectContaining(expectedData))
+    })
+  })
+
+  describe('prepareConcessionsData', () => {
+    it('should add senior concession if the licensee is senior', async () => {
+      const samplePermission = existingPermission()
+      samplePermission.licensee.birthDate = '1954-01-01'
+      const ppd = await preparePermissionDataForRenewal(samplePermission)
+      const senior = { name: 'Senior', id: 'd0ece997-ef65-e611-80dc-c4346bad4004', proof: { type: 'No Proof' } }
+      expect(ppd.concessions[0]).toEqual(senior)
+    })
+
+    it('should remove senior concession if the licensee is not senior', async () => {
```

Comment:
Technically this isn't removing a concession as none exists in the first place – can we rename the test to reflect this?